src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -68,7 +68,5 @@ public void testDoGet_UserDoesNotExist() throws IOException, ServletException {
 
         profileServlet.setUserStore(mockUserStore);
         profileServlet.doGet(mockRequest, mockResponse);
-        Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
```

Comment:
Hmm if you remove this the whole test doesn't do anything, can you not formulate a valid test?

Reply:
If the user doesn't exist, I could send an empty string and forward to profile.jsp so it can display the error message "User does not exist" on the web page, so then I could keep the Mockito.verify statement for the test?

Reply:
^ Might go back to the earlier version where display error in web page rather than terminal.